apps/app/src/views/routes/Settings/components/profile/ProfileEditForm.tsx

```diff
@@ -0,0 +1,431 @@
+import React from 'react';
+import {
+  HStack,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  Alert,
+  AlertIcon,
+  VStack,
+  Text,
+  Button,
+  ModalFooter,
+  FormControl,
+  FormLabel,
+  FormErrorMessage,
+  Input
+} from '@chakra-ui/react';
+import * as yup from 'yup';
+import { rolesSchema } from '../utils/Roles';
+import { useMutation } from '@apollo/client';
+import { usePhoton } from '@photonhealth/react';
+import { FragmentType, graphql, useFragment } from 'apps/app/src/gql';
+import { FormikStateSelect, yupStateSchema } from '../utils/States';
+import { FormikTouched, FormikErrors, ErrorMessage, Field, Formik } from 'formik';
+import { Role } from 'packages/sdk/dist/types';
+import { userFragment } from '../utils/UserFragment';
+
+interface EditProfileActionProps {
+  user?: FragmentType<typeof userFragment>;
```

Comment:
This should be `user: FragmentType<blah>`. Its always required and non-nullable

Reply:
done